app/scenes/Login/Notices.tsx

```diff
@@ -18,6 +18,13 @@ export default function Notices() {
           invite email.
         </NoticeAlert>
       )}
+      {notice === "gmail-account-creation" && (
+        <NoticeAlert>
+          Sorry, a new account cannot be created with a personal Gmail address.
```

Comment:
```suggestion
          Sorry, a new team cannot be created with a personal Gmail address.
```

Maybe clearer, we do allow user accounts after all in the case of an invite

Reply:
I think the current language has less of a tendency to confuse. There isn't really a point in the process where a user thinks "I'm creating a team".

They're either signing up or logging in. If they've been invited, it's more of a logging in concept than account creation.